app/assets/javascripts/summernote-init.js

```diff
@@ -20,7 +20,7 @@ $(document).on('ready', function() {
       height: 150,
       toolbar: [
         ["style", ["bold", "italic", "underline", "clear"]],
-        ["table", ["table"]],
+        ["insert", ["table", "picture", "link", "video"]],
```

Comment:
will this affect any views that should NOT have picture, link, and video?

Reply:
As far as I'm aware, they should all have all that, but there are two separate summernote initializers and I believe this one is specific to loan questions.